src/ip.cpp

```diff
@@ -717,11 +716,12 @@ void zmq::assert_socket_tuning_error (zmq::fd_t s_, int rc_)
         err = errno;
     if (err != 0) {
         errno = err;
-        errno_assert (errno == ECONNREFUSED || errno == ECONNRESET
-                      || errno == ECONNABORTED || errno == EINTR
-                      || errno == ETIMEDOUT || errno == EHOSTUNREACH
-                      || errno == ENETUNREACH || errno == ENETDOWN
-                      || errno == ENETRESET || errno == EINVAL);
+        errno_assert (
+          errno == ECONNREFUSED || errno == ECONNRESET || errno == ECONNABORTED
+          || errno == EINTR || errno == ETIMEDOUT || errno == EHOSTUNREACH
+          || errno == ENETUNREACH || errno == ENETDOWN || errno == ENETRESET
+          || errno == EINVAL || errno == EADDRINUSE || errno == EACCES
+          || errno == EWOULDBLOCK || errno == EAGAIN || errno == EPIPE);
```

Comment:
Please do not add new errors mixed in with refactoring. It needs to be carefully weighted and tested to avoid introducing deadlocks.

Reply:
Ok, sure! So did unittest_poller keep running because I made some errors non-critical?

Reply:
Not sure - should be investigated